lms/templates/instructor/instructor_dashboard_2/send_email.html

```diff
@@ -37,7 +36,7 @@
   <li class="field">
     <label>${_("Message:")}</label>
       <div class="email-editor">
-      ${ HTML(section_data['editor']) }
+      ${section_data['editor']}
```

Comment:
@douglashall: FYI: I may be changing the recommendation for this such that this call to HTML() is moved into the Python code where the HTML is actually getting generated.  See comment above for details.